### PR TITLE
corectl 0.7.7

### PR DESCRIPTION
**Pull request by AntonioMeireles**
Hi again!

please bump 😄 

Thanks in advance

### CODE REVIEW

File: Formula/corectl.rb [hunk 1]
```diff
@@ -4,13 +4,12 @@ class Corectl < Formula
   head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
 
   stable do
-    url "https://github.com/TheNewNormal/corectl/archive/v0.7.0.tar.gz"
-    sha256 "bdbb7d453232995c93862b7e97edb43d312f4976c84c3781e902a140c88ebb45"
-
-    # until 0.7.1 is out
+    url "https://github.com/TheNewNormal/corectl/archive/v0.7.6.tar.gz"
+    sha256 "a2c60b3dff3fa28f0829453c59d27b5cff82e76f7b994f10d35d1e4f6ecabdd4"
+    # until 0.7.6 is out
```
Comment (DomT4) [inline Formula/corectl.rb]: The version you've bumped to is `0.7.6` 😄

**Comment by AntonioMeireles** [comment]
@DomT4 addressed.

**Comment by AntonioMeireles** [comment]
meanwhile i got 0.7.7 out...

File: Formula/corectl.rb [hunk 2]
```diff
@@ -4,14 +4,8 @@ class Corectl < Formula
   head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
 
   stable do
```
Comment (ilovezfs) [inline Formula/corectl.rb]: the `stable do ... end` block also needs to be removed, and the url and sha256 moved all the way up in the formula above the head url.

**Comment by AntonioMeireles** [comment]
@ilovezfs addressed, hopefully

**Comment by ilovezfs** [comment]
@AntonioMeireles Yup, thank you. Another question while we're at it again! Is "godep" currently actually used by the formula? I notice you now have a vendor folder in your repository.

**Comment by AntonioMeireles** [comment]
yes it is. see https://github.com/tools/godep#migrating-to-vendor for the gory details.

**Comment by ilovezfs** [comment]
@AntonioMeireles Hmmm. I thought that when you used that procedure to migrate to the vendor folder, that godep could be dropped entirely.

**Comment by AntonioMeireles** [comment]
@ilovezfs no. no, no. `Godeps/Godeps.json` is still where is has been always. this is what `godep` consumes. it's just that the actual payload moved from `Godeps/_src/workspace/` to `vendor/`

Also do note that `godep` is used in the Makefile (L14:15) to help assemble the GOPATH

**Comment by ilovezfs** [comment]
I see. Usually, we would do this (I left out the ldflags):
```
ENV["GOPATH"] = buildpath
(buildpath/("src/github.com/TheNewNormal/corectl").install buildpath.children
cd "src/github.com/TheNewNormal/corectl" do
  go build -o server cmd/server/main.go
  go build -o client cmd/client/main.go
end
```

**Comment by AntonioMeireles** [comment]
ok, thanxs. will look at that once not in vacation .. anyway 'fatness' atm only at build time, not runtime.

**Comment by ilovezfs** [comment]
@AntonioMeireles yup, no problem. Just thought I'd point out that `go build` actually does work out of the box for your software (I just tried) once that vendor folder is there.

**Comment by ilovezfs** [comment]
@AntonioMeireles Thanks for keeping this formula up to date with your software! You're awesome :100:

**Comment by DomT4** [comment]
> I thought that when you used that procedure to migrate to the vendor folder, that godep could be dropped entirely.

Depends how you do it. Godep does support the Vendor experiment. It's not my preferred way of Go vendoring, but it is valid 🙈.

**Comment by ilovezfs** [comment]
@DomT4 My point is that he already did it! https://github.com/TheNewNormal/corectl/tree/golang/vendor

So we might as well reap the benefits ...

**Pull request closed without merging**